recipe/meta.yaml

```diff
@@ -27,7 +28,7 @@ requirements:
     - python
     - numpy >=1.17.2
     - packaging  # hotfix for utils, can be dropped with lit-utils >=0.5
-    - pytorch >=1.8.1
+    - pytorch >=1.8.1,<2.0.0
```

Comment:
I'll fix it as py==311

Reply:
it's not necessary. I think as soon as [PR-5](https://github.com/AnacondaRecipes/torchmetrics-feedstock/pull/5) builds (waiting on `pytorch 2.0.*`) is building I will rebase my branch on this. Or you will rebase on top of mine, in case mine is merged first (I doubt it).

Reply:
`torchmetrics` still isn't compatible with pytorch 2.x.x ,see https://github.com/Lightning-AI/torchmetrics/blob/54a06013cdac4895bf8e85b583c5f220388ebc1d/requirements.txt#L5

Reply:
ok, then I will rebase for openssl support

Reply:
@skupr-anaconda I think the upper bound is for their CI only. When I download their wheels and look at the metadata, I see `Requires-Dist: torch (>=1.8.1)`. See https://inspector.pypi.io/project/torchmetrics/0.11.4/packages/fb/47/6e9f9b41c48750a45ad07cc6d43a2979bfc09e6989656aece97cc59cbef1/torchmetrics-0.11.4-py3-none-any.whl/torchmetrics-0.11.4.dist-info/METADATA.

Reply:
Ah, yes, you are right

Reply:
Fixed